vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/codec/json/JsonObjectCodec.java

```diff
@@ -2,11 +2,10 @@
 
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
-import org.bson.BsonReader;
-import org.bson.BsonString;
-import org.bson.BsonValue;
+import org.bson.*;
```

Comment:
nit: Probably don't want the wildcard import (seems to go against the overall code style).